README.md

```diff
@@ -157,16 +152,23 @@ Before you get started, **make sure you have read the [Git branch setup instruci
 
 To view the documentation including examples, contracts, component status, and to add functionality or fix issues locally, you can:
 
-1. `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
-2. `npm install`
-3. `npm start`
+1.  `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
+2.  `npm install` _For Windows developers, please read the next section before running this_
+3.  `npm start`
 
 This will start a demo page from the office-ui-fabric-react package folder, which will open a web browser with the example page. You can make changes to the code which will automatically build and refresh the page using live-reload.
 
 To build and run tests for all packages in the repo, you can run `npm run build` from the root.
 
 To build individual packages within the `packages/*/` folders, you can use `npm run build` in each individually. Note that because the packages are symlinked together, you must manage building dependencies in the right order, or use the `rush` tool to build to the specific package you want. (See advanced tips below.)
 
+## Note for Windows Users
+
+Due to a known bug in #5174, Windows users who are using npm 6.1.0+ should use one of the following workaround:
+
+1.  downgrade npm to npm 5.6.0 - `npm i -g npm@5.6.0`
+2.  instead of `npm install`, use `rush install && rush build`

```

Comment:
Not using npm install even once means

1) They need to install rush globally
2) They won't have the precommit hooks running (prettier)

Since prettier is not enforced at CI, this means we'll start having code with inconsistent styling going in...

Reply:
Yes. Please can we not suggest using rush directly? If we need to abstract in an npm script, let's do that. It makes it much easier to change out how we use rush, or change out rush altogether if need be.

Reply:
Why are people still using npm 6? Could we just suggest a minbar that isn't something so old?

Reply:
We can get rid of the 2nd suggestion. And enforce people to use npm 5.6.0 with "engines" in our package.json.